lib/discourse_automation/scripts/user_group_membership_through_badge.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+DiscourseAutomation::Scriptable::USER_GROUP_MEMBERSHIP_THROUGH_BADGE =
+  "user_group_membership_through_badge"
+DiscourseAutomation::Scriptable::USER_GROUP_MEMBERSHIP_THROUGH_BADGE_BULK_MODIFY_START_COUNT = 1000
```

Comment:
🤷🏾  Arbitrary limit alert.  I wonder if this is even needed, the `add_user_to_group_through_custom_field ` script doesn't do this at the moment.

Better safe than sorry, we probably don't want to be doing  1000s of `Group#add`  calls anyways